### DISCUSSION TITLE
Add a way to set gravity vector for RigidBody

### DISCUSSION THREAD

**Discussion by me2beats** [godotengine/godot-proposals#4747] [Engine Core]
I'm trying to create a driving a car on the planet.
And this doesn't look easy.
Imo this would be handy to be able to set gravity direction Vector3(x,y,z).
Could be a property or a method of RigidBody

**Comment by theraot** [comment]
You can use an Area to set or modify the gravity for any RigidBody inside of it, including using a gravity point.

**Reply by me2beats** [reply]
oh sounds interesting, I'll try it
